docs/standard/native-interop/pinvoke.md

```diff
@@ -16,12 +16,13 @@ Let's start from the most common example, and that is calling unmanaged function
 The previous example is simple, but it does show off what's needed to invoke unmanaged functions from managed code. Let's step through the example:
 
 - Line #2 shows the using statement for the `System.Runtime.InteropServices` namespace that holds all the items needed.
-- Line #8 introduces the `DllImport` attribute. This attribute tells the runtime that it should load the unmanaged DLL. The string passed in is the DLL our target function is in. Additionally, it specifies which [character set](./charset.md) to use for marshalling the strings. Finally, it specifies that this function calls [SetLastError](/windows/desktop/api/errhandlingapi/nf-errhandlingapi-setlasterror) and that the runtime should capture that error code so the user can retrieve it via <xref:System.Runtime.InteropServices.Marshal.GetLastWin32Error?displayProperty=nameWithType>.
-- Line #9 is the crux of the P/Invoke work. It defines a managed method that has the **exact same signature** as the unmanaged one. The declaration has a new keyword that you can notice, `extern`, which tells the runtime this is an external method, and that when you invoke it, the runtime should find it in the DLL specified in `DllImport` attribute.
+- Line #8 introduces the `LibraryImport` attribute. This attribute tells the runtime that it should load the unmanaged DLL. The string passed in is the DLL our target function is in. Additionally, it specifies which [character set](./charset.md) to use for marshalling the strings. Finally, it specifies that this function calls [SetLastError](/windows/desktop/api/errhandlingapi/nf-errhandlingapi-setlasterror) and that the runtime should capture that error code so the user can retrieve it via <xref:System.Runtime.InteropServices.Marshal.GetLastPInvokeError?displayProperty=nameWithType>.
```

Comment:
We should make decision on whether we want to continue using the term "unmanaged".

Reply:
We do. Note two modern .NET interop APIs - [`UnmanagedCallersOnlyAttribute`](https://learn.microsoft.com/dotnet/api/system.runtime.interopservices.unmanagedcallersonlyattribute) and [`UnmanagedCallConvAttribute`](https://learn.microsoft.com/dotnet/api/system.runtime.interopservices.unmanagedcallconvattribute). It is clearer than saying "native" or "C/C++".